<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VcsBase::Internal::NickNameDialog</class>
 <widget class="QDialog" name="VcsBase::Internal::NickNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nicknames</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Utils::FancyLineEdit" name="filterLineEdit"/>
   </item>
   <item>
    <widget class="Utils::TreeView" name="filterTreeView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::TreeView</class>
   <extends>QTreeView</extends>
   <header location="global">utils/itemviews.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VcsBase::Internal::NickNameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VcsBase::Internal::NickNameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
